httpcore5/src/main/java/org/apache/hc/core5/util/Args.java

```diff
@@ -167,6 +167,11 @@ public static long positive(final long n, final String name) {
         return n;
     }
 
+    public static <T extends TimeValue> T positive(final T timeValue, final String name) {
+        positive(timeValue.toMilliseconds(), name);
```

Comment:
This can be dangerous because if someone supplies microsecond resolution it will be 0 ms, thus will trigger an exception. You should rather use `getDuration()`.

Reply:
Done.